docs/src/use/migrate-to-9.0.0.md

```diff
@@ -432,7 +433,13 @@ class SomeClass {
 
 **To address:** Fix the problems reported by the rule or revert to the previous behavior by setting the `enforceForClassMembers` option to `false`.
 
-**Related issue(s):** [#18042](https://github.com/eslint/eslint/issues/18042)
+## <a name="camelcase"></a> `camelcase` allow option only accepts an array of strings
```

Comment:
```suggestion
**Related issue(s):** [#18042](https://github.com/eslint/eslint/issues/18042)

## <a name="camelcase"></a> `camelcase` allow option only accepts an array of strings
```

Seems that the line for a related issue for the previous item was accidentally deleted.

Reply:
arg, shit, fixing